orders/views.py

```diff
@@ -74,4 +75,31 @@ def get(self, request):
                 order__order_status_id=OrderStatus.BASKET
             )]
 
-        return JsonResponse({'message':'SUCCESS', 'items_in_cart':order_items}, status=200)
\ No newline at end of file
+        return JsonResponse({'message':'SUCCESS', 'items_in_cart':order_items}, status=200)
+
+    @login_decorator
+    def patch(self, request):
+        try:
+            data          = json.loads(request.body)
+            order_item_id = data['order_item_id']
+            count         = data.get('count')
+            select        = data.get('select')
+    
+            if (type(count) != int or count < 0) and count != None:
+                return JsonResponse({'message':'INVALID_COUNT_TYPE'}, status=400)
+            
+            if not OrderItem.objects.filter(order__user=request.user, id=order_item_id).exists():
+                return JsonResponse({'message':'INVALID_ORDER_ITEM'}, status=400)
+            
+            order_item = OrderItem.objects.filter(id=order_item_id)
```

Comment:
```suggestion
            order_item = OrderItem.objects.get(id=order_item_id)
```
get을 이용해서 데이터를 특정 짓는 것이 더 좋습니다~